var/spack/repos/builtin/packages/py-pyfr/package.py

```diff
@@ -31,21 +35,24 @@ class PyPyfr(PythonPackage):
     variant("scotch", default=False, description="Scotch for mesh partitioning")
     variant("cuda", default=False, description="CUDA backend support")
     variant("hip", default=False, description="HIP backend support")
+    variant("libxsmm", default=True, description="LIBXSMM for OpenMP backend")
+    variant("scipy", default=True, description="Scipy acceleration for point sampling")
 
     # Required dependencies
     depends_on("python@3.9:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     depends_on("py-gimmik@2.2:2", type=('build', 'run'))
+    depends_on("py-gimmik@2.3:2", when='@1.14.0:', type=('build', 'run'))
```

Comment:
This is implied by the line before it

Reply:
Thanks for catching that, I think this is what you meant:
`depends_on("py-gimmik@2.3:2", type=('build', 'run'))`

Reply:
Oops, didn't notice the change from 2.2 to 2.3. Can you actually reverse the last commit? I think I'm going blind 😆

Reply:
@adamjstewart, Done